src/components/Header.js

```diff
@@ -39,12 +39,12 @@ export const HeaderToolbar = () => {
                 <Navbar bg="dark" variant="dark" className="text-center"  >
                     {/*<Navbar.Brand href="/home">Crypto Pro</Navbar.Brand> */}
                     {/* Crypto Pro header click -> home  */}
-                    <Navbar.Brand className="p-3" href="/">Crypto Pro</Navbar.Brand>
+                    <Navbar.Brand href="/home">Crypto Pro</Navbar.Brand>
```

Comment:
We need the padding so it doesn't touch the sides of the header. Also "/" is home so this line doesn't need to be changed.

Reply:
- The padding has been re-added.
- The `/home` path was required to get the hosted site to resolve to the home page when the Logo is clicked.